Main/ClientApp/src/Components/Drawer.tsx

```diff
@@ -32,13 +33,21 @@ const useStyles = makeStyles((theme: Theme) =>
       color: "black",
       textDecoration: "none"
     },
+    addItem: {
+      paddingLeft: "15px"
+    },
     // necessary for content to be below app bar
     toolbar: theme.mixins.toolbar
   })
 );
 
 export default function Drawer() {
   const classes = useStyles();
+  const [dialogOpen, setDialogOpen] = React.useState(false);
+
+  const toggleAddComponentDialogOpen = () => {
+    setDialogOpen(!dialogOpen);
```

Comment:
AddComponent?

Reply:
collection... mistakes like this happens when you coding in the middle of the night